### ISSUE TITLE
Cleaner handling of FreeIPA requirement

### ISSUE THREAD

**Issue by sgallagher** [fleet-commander/fc-admin#159]
Right now, if I install the Fleet Commander app on Cockpit, it installs cleanly but then the application reads:
```
Error connecting to IPA server
Error connecting to IPA server. Check system logs for details
```
It gives no more information and doesn't help identify what the user needs to do to fix the problem.

After discussing with @aruiz, @olivergs and @mvollmer on IRC, it's clear that what is needed is:
1. The system must be enrolled in a FreeIPA domain.
1. The FreeIPA server for the domain must be running the `freeipa-desktop-profile` plugin.
1. The logged-in user must be a FreeIPA user account with sufficient privilege to manage the desktop profiles.

My recommendation is that the application should test for each of these conditions and provide feedback to the user about which problem is preventing access. Testing for enrollment in a FreeIPA domain can be done with `realmd`, which I believe already has interfaces for it in Cockpit. Testing for the presence of the desktop-profile plugin may be more complicated, but I trust that the Fleet Commander developers probably can figure it out. The privilege question can probably be handled the same way the rest of Cockpit does, with permissions evaluated per-action rather than per-component.

**Comment by olivergs** [comment]
I agree we need to give more information to users on connection errors to FreeIPA server.

The three things shoud be added every time you open Fleet Commander Admin into cockpit, so it is on our side of things to develop that enhancements.

Keeping this issue open for planning it. Thanks @sgallagher
